src/main/java/org/folio/consortia/service/impl/UserServiceImpl.java

```diff
@@ -64,7 +60,7 @@ public void deleteById(String userId) {
   }
 
   public User prepareShadowUser(UUID userId, String tenantId) {
-    try (var context = new FolioExecutionContextSetter(prepareContextForTenant(tenantId, folioModuleMetadata, (FolioExecutionContext) folioExecutionContext.getInstance()))) {
+    try (var context = new FolioExecutionContextSetter(contextHelper.getFolioExecutionContext(tenantId))) {
```

Comment:
we don't need to use system user in all places, only in TenantService when initiating the tenant

Reply:
Fixed